core/src/main/java/org/apache/accumulo/core/fate/ReadOnlyFateStore.java

```diff
@@ -57,8 +54,7 @@ enum TStatus {
     /** Transaction that is eligible to be executed */
     SUBMITTED;
 
-    public static final Set<TStatus> ALL_STATUSES =
-        Arrays.stream(values()).collect(Collectors.toUnmodifiableSet());
+    public static final EnumSet<TStatus> ALL_STATUSES = EnumSet.allOf(TStatus.class);
```

Comment:
Since you can't make it immutable, it's better to just not have this. The `EnumSet.allOf(TStatus.class)` can be put inline in the few places where it is needed. Yes, it will be computed each time... but it's fast and efficient, and it's the only way to be safe that it hasn't been modified.

If you want to make it ever so slightly more convenient, you could replace this static set with a static generator method, as in:

```suggestion
    public static EnumSet<TStatus> allOf() {
      return EnumSet.allOf(TStatus.class);
    }
```

Then, you can call `TStatus.allOf()` instead of `EnumSet.allOf(TStatus.class)`, but I don't think this is saving much, and isn't really that useful.